lib/scope.js

```diff
@@ -593,32 +595,42 @@ module.exports = {
 
     // Get sought var
     let encoded_sought_var = $($( `input[name="_event"]` )).attr( `value` );
+    // TODO: Change to `sought_name`?
     let sought_var_str = await scope.fromBase64( scope, { base64_str: encoded_sought_var });
-    page_data.sought_var = JSON.parse( sought_var_str );  // Why is this a list? Can multiple vars be sought?
+    page_data.sought_var = JSON.parse( sought_var_str )[0];  // Why is this a list? Can multiple vars be sought?
     // Some `show if` fields have `names` like '_field_n' instead of their var name. Ex: _field_1
     let field_like_names = await scope.getFieldNamesDict( scope, { $: $ });
 
     // Will deal with `option` once inside `select`
-    let all_nodes = $( `canvas, fieldset button, .daquestionactionbutton, .form-group input, .form-group select, .form-group textarea` );
-
-    page_data.fields = [];
+    let all_nodes = $( `#dasigpage, fieldset button, .daquestionactionbutton, .form-group input, .form-group select, .form-group textarea` );
+    let proxies_to_names_map = {};
+    let temp_fields = [];
     for ( let node of all_nodes ) {
       // Decsion: Do not abstract the below. There's too much data to pass around for it to make sense
-
       let $node = $( node );
-      let field_data = { selector: '', tag: node.name, rows: [], type: '' };
+      let field_data = {
+        node: node,
+        selector: '',
+        values: [],
+        var_names: [],
+        name_patterns: [],
```

Comment:
Might be useful to document somewhere what these three values mean. The `field_data` structure seems pretty important for our testing framework, so it would be good to have documentation about what it should contain.

Reply:
#194!